docs/quickstart-docker.md

```diff
@@ -0,0 +1,143 @@
+# Start a Docker container and take a backup
+
+Percona XtraBackup works in combination with a MySQL-compatible database, Percona Server for MySQL, in our scenario. To use Percona XtraBackup, you need to run it in its own Docker container and connect it to the Percona Server for MySQL container. As soon as you [start Percona Server for MySQL in a docker container](https://docs.percona.com/percona-server/8.0/quickstart-docker.html) and [create a database and table in Percona Server for MySQL](https://docs.percona.com/percona-server/8.0/quickstart-database.html), you are ready to backup your databes with Percona XtraBackup.
```

Comment:
To use Percona XtraBackup, you need to run it in its own Docker container and connect it to the Percona Server for MySQL container.

too many "it" - rewrite to be clear what "it" refers to

Reply:
done